source/classes.tex

```diff
@@ -6265,8 +6261,8 @@
 of constructors and destructors for the object declared by the
 \grammarterm{exception-declaration}.
 \begin{note}
-There cannot be a move from the exception object because it is
-always an lvalue.
+Since the exception object is always an lvalue,
+it is never moved from, which would potentially be incorrect.
```

Comment:
"would" is also bad in notes, I think.

Reply:
I think "would" is fine, actually. What isn't allowed is the use of "would" to express a provision, but we're not doing that here.